src/components/tab-group/tab-group.component.ts

```diff
@@ -434,7 +468,7 @@ export default class SlTabGroup extends ShoelaceElement {
               `
             : ''}
 
-          <div class="tab-group__nav">
+          <div class="tab-group__nav"  @scrollend=${this.updateScrollButtons}>
```

Comment:
This isn't supported in Safari yet, so we'll need to use @alenaksu's scrollend polyfill. Carousel uses it. Just add this to the top of the file and it should work in Safari.

```ts
import '../../internal/scrollend-polyfill.js';
```